enhancements/etcd/automated-backups-no-config.md

```diff
@@ -0,0 +1,242 @@
+---
+title: automated-backups-no-config
+authors:
+  - "@elbehery"
+reviewers:
+  - "@soltysh"
+  - "@dusk125"
+  - "@hasbro17"
+  - "@tjungblu"
+  - "@williamcaban"
+approvers:
+  - "@soltysh"
+  - "@dusk125"
+  - "@hasbro17"
+  - "@tjungblu"
+  - "@williamcaban"
+api-approvers:
+  - "@soltysh"
+creation-date: 2024-06-17
+last-updated: 2024-06-17
+tracking-link:
+  - https://issues.redhat.com/browse/ETCD-609
+see-also:
+  - "https://issues.redhat.com/browse/OCPSTRAT-1411"
+  - "https://issues.redhat.com/browse/OCPSTRAT-529"
+---
+
+
+# Automated Backups of etcd with No Config
+
+## Summary
+
+Enable the automated backups of etcd from day 1 without provided config from the user.
+
+## Motivation
+The [current automated backup of etcd](https://docs.openshift.com/container-platform/4.15/backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.html#creating-automated-etcd-backups_backup-etcd) of an OpenShift
+cluster relies on user provided configurations. 
+
+To improve the customers experience, this work proposes taking etcd backup using default config from day one, without additional configuration from the user.
+
+
+### Goals
+- Backups should be taken without configuration after cluster installation from day 1.
+- Backups are saved to a default PersistentVolume, that could be overridden by user.
+- Backups are taken according to a default schedule, that could be overridden by user.
+- Backups are taken according to a default retention policy, that could be overridden by user.
+
+
+### Non-Goals
+- Save cluster backups to cloud storage e.g S3.
+  - This could be a future enhancement or extension to the API.
+- Automate cluster restoration.
+- Provide automated backups for non-self hosted architectures like Hypershift.
+
+### User Stories
+- As a cluster administrator I want cluster backup to be taken without configuration.
+- As a cluster administrator I want to schedule recurring cluster backups so that I have a recent cluster state to recover from in the event of quorum loss (i.e. losing a majority of control-plane nodes).
+- As a cluster administrator I want to have failure to take cluster backups for more than a configurable period to be reported to me via critical alerts.
+
+
+## Proposal
+- Provide a default etcd backup configuration as a `EtcdBackup` CR.
+- Provide a default reliable storage mechanism to save the backup data on.
+- Provide a separate feature gate for the automated backup with default config (under discussion).
+
+
+### Workflow Description
+- The user will enable the AutomatedBackupNoConfig feature gate (under discussion).
```

Comment:
Given that `AutomatedEtcdBackup` is still a tech preview feature, I'd consider expanding that functionality with the default configuration, rather than adding another one. This will save a lot of problems for some of the fields in the API you're planning to provide default values for.

Reply:
+1, I will fix this 👍🏽